spec/requests/api/v1/endpoints/merchants_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe 'Merchants API' do
+  it 'returns a list of merchants' do
+    merchant1, merchant2, merchant3 = create_list(:merchant, 3)
+    name = merchant1.name
+
+    get '/api/v1/merchants'
+    merchants = JSON.parse(response.body)
+    merchant = merchants.first
+
+    expect(response).to be_success
+    expect(merchants.count).to eq 3
+    expect(merchant['name']).to eq name
```

Comment:
Excellent - we're checking to make sure we get merchants back in the index without going overboard.